com.ekonomikas.merkato.json

```diff
@@ -0,0 +1,61 @@
+{
+    "app-id" : "com.ekonomikas.merkato",
```

Comment:
the domain `ekonmikas.com` seems to be parked. Is it yours (or upstream) ?

Reply:
The domain belongs to me.

Reply:
ok